packages/react-admin/src/mui/list/BulkActions.js

```diff
@@ -0,0 +1,109 @@
+import React, { cloneElement, Children, Component } from 'react';
+import PropTypes from 'prop-types';
+import MoreVertIcon from 'material-ui-icons/MoreVert';
+import Menu from 'material-ui/Menu';
+import classnames from 'classnames';
+
+import Button from '../button/Button';
+import translate from '../../i18n/translate';
+import BulkDeleteMenuItem from './BulkDeleteMenuItem';
+
+const sanitizeRestProps = ({
+    basePath,
+    filterValues,
+    resource,
+    onUnselectItems,
+    ...rest
+}) => rest;
+
+class BulkActions extends Component {
+    state = {
+        anchorElement: null,
+    };
+
+    handleClick = event => {
+        this.setState({ anchorElement: event.currentTarget });
```

Comment:
Storing an element in the state doesn't smell good. It feels like you should use a `ref` here, and store `isOpen` in the state instead.